yarn.lock

```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@^7.0.0-beta.35":
   version "7.0.0-beta.52"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0-beta.52.tgz#192483bfa0d1e467c101571c21029ccb74af2801"
-  integrity sha1-GSSDv6DR5GfBAVccIQKcy3SvKAE=
```

Comment:
@mnajdova can you please update your `yarn` to a version higher than `1.10.0`; we don't want to remove these lines; see #437 for details

Reply:
Done, thanks!